core/justfile

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env just --justfile
+
+gradle := if os() == "windows" { "gradlew.bat" } else { "./gradlew" }
+javaflags := "-ea -Xmx12g"
+
+# Display available recipes
+help:
+    @just --list --unsorted
+
+# Run the application; does not start other services
+run:
+    ALL_INFRA=true java {{javaflags}} -jar build/libs/osrd-all.jar worker --editoast-url http://localhost:8090/
```

Comment:
With the scalability and core containers being spawned automatically, it's sadly not quite as simple as that.

When we just want to run a single worker process (the most common use case when running locally), we need to add the env variable `ALL_INFRA=TRUE`. But we also need to run the rest of the stack in "single worker" mode: `sw` flag in `osrd-compose`, or the env variables set in [docker-compose.single-worker.yml](https://github.com/OpenRailAssociation/osrd/blob/dev/docker/)

Reply:
I added a comment in https://github.com/OpenRailAssociation/osrd/pull/11174/commits/3691d77d4980ac87c925113c4e096892e2b3461b